testing/a11y/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "a11y",
+  "version": "0.0.0",
+  "private": true,
+  "type": "module",
+  "scripts": {
+    "test": "cypress run --component",
+    "open": "cypress open --component",
+    "clean": "rimraf .turbo && rimraf node_modules"
+  },
+  "dependencies": {
+    "@itwin/itwinui-react": "3.0.0-dev.3",
+    "axe-core": "^4.7.2",
+    "cypress": "^12.14.0",
```

Comment:
Looks like CI is failing. This could potentially be because of multiple versions of cypress being installed (one in storybook).

Could you change all of them to be `12.16.0` and run `yarn`?

Files you need to change:
1. `apps/storybook/package.json`
2. `apps/storybook/scripts/run-tests.js`
3. `testing/a11y/package.json`

Reply:
Done